src/api/auth.ts

```diff
@@ -589,6 +589,27 @@ export function userHasWritePermissions(
     });
 }
 
+export function deletePackage(pkg: string, user: string, auth: any): Promise<any> {
+  return new Promise((resolve, reject) => {
+    userHasWritePermissions(user, pkg, auth)
+      .then(permission => {
+        if (permission) {
+          let pkgIndex = auth.packages.findIndex(p => p.name === pkg);
+          if (pkgIndex !== -1) {
+            auth.packages.splice(pkgIndex, 1);
```

Comment:
you only remove package from `auth.json` but in local storage there is still present I believe?